puphpet/config.yaml.template

```diff
@@ -147,15 +174,41 @@ php:
     composer_home: ''
     modules:
         php:
-            - cli
-            - intl
-            - mcrypt
-            - memcached
-            - sqlite
-            - curl
+            - cli # Needed for shells
+            #- curl
+            #- gd
+            #- geoip
+            #- imagick
+            - intl # Required by Cake 3.
+            - mbstring # Required by Cake 3.
+            - mcrypt # Required by Cake 3.
+            #- memcache
+            #- memcached
+            #- mysql
+            #- readline
+            #- sqlite
```

Comment:
See this is what I was talking about. [sigh]

The skeleton is configured to run tests in mysql already, so sqlite isn't actually _necessary_. If we want to switch the default DB for tests, then the bundled database configs would need to be updated to reflect that too.

Also, sqlite has caused compatibility issues for us more than once. I don't argue that it's probably faster the majority of the time, but mysql is safer because it exactly matches the app. I think developers should **have** to opt into this if their project's queries allow them to.